rxredux/build.gradle.kts

```diff
@@ -44,9 +44,9 @@ task<JacocoReport>("codeCoverageReport") {
 
     val mainSrc = "${project.projectDir}/src/main/java"
 
-    setSourceDirectories(files(mainSrc))
```

Comment:
`setSourceDirectories`, `setClassDirectories`, `setExecutionData` are deprecated and recommend to replace them with `sourceDirectories.setFrom`, `classDirectories.setFrom`, `executionData.setFrom` respectively